developer/src/kmc-keyboard/src/compiler/compiler.ts

```diff
@@ -98,7 +98,7 @@ export default class Compiler {
       // TODO-LDML: The unboxed data doesn't match the schema anymore. Skipping validation, for now.
 
       // try {
-      //   if (!reader.validate(source, this.callbacks.loadLdmlKeyboardTestSchema())) {
+      //   if (!reader.validate(source, this.callbacks.loadSchema('ldml-keyboard-test'))) {
```

Comment:
Should this be `ldml-keyboardtest`?

reader-callback-test.ts l. 24

Reply:
Good catch; it's commented out code for now so I didn't get compiler error on it. I will leave it for now because it's an easy fix when we uncomment the code (and saves another build run)